tests/ManiaTemplates.Tests/Components/MtComponentAttributesTest.cs

```diff
@@ -0,0 +1,24 @@
+﻿using ManiaTemplates.Components;
+
+namespace ManiaTemplates.Tests.Components;
+
+public class MtComponentAttributesTest
+{
+    private readonly MtComponentAttributes _mtComponentAttributes = new();
+
+    [Fact]
+    public void Should_Contain_Single_Attribute_Pair()
+    {
+        _mtComponentAttributes["key"] = "value";
+        Assert.Single(_mtComponentAttributes);
+    }
+
+    [Fact]
+    public void Should_Remove_Existing()
+    {
+        var result = _mtComponentAttributes.Pull("key");
```

Comment:
This test will fail, as the construcutor was removed. 

```suggestion
        _mtComponentAttributes["key"] = "value";
        var result = _mtComponentAttributes.Pull("key");
```